main.go

```diff
@@ -64,21 +65,15 @@ func main() {
 		buildDate = time.Now()
 	}
 
-	var net string
-	switch cfg.Network {
-	case "testnet":
-		net = "testnet3"
-	default:
-		net = cfg.Network
-	}
-
-	logDir := filepath.Join(cfg.LogDir, net)
-	assetsManager, err := libwallet.NewAssetsManager(cfg.HomeDir, logDir, net)
+	logDir := filepath.Join(cfg.LogDir, string(cfg.net))
+	assetsManager, err := libwallet.NewAssetsManager(cfg.HomeDir, logDir, cfg.net)
 	if err != nil {
 		log.Errorf("init assetsManager error: %v", err)
 		return
 	}
 
+	assetsManager.MaxLogZips = cfg.MaxLogZips
```

Comment:
This is only used in UI, not by assetsmanager. Put it in load, for now. I'm not sure the `AppInfo` struct is in master yet, but if it is, that's a better place to put it.

Reply:
Okay.